python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml

```diff
@@ -0,0 +1,23 @@
+name: dining_prefs
+file_type: .csv
+author: J.L. Moreno
+url:  https://data.rapids.ai/cugraph/datasets/dining_prefs.csv
+refs:
+  J. L. Moreno (1960). The Sociometry Reader. The Free Press, Glencoe, Illinois, pg.35
+delim: " "
+header: None
+col_names:
+  - src
+  - dst
+  - wgt
+col_types:
+  - string
+  - string
+  - int
+has_loop: false
+is_directed: false
+is_multigraph: false
+is_symmetric: true
+number_of_edges: 42
+number_of_nodes: 26
+number_of_lines: 52
```

Comment:
We got rid of `number_of_lines` when promoting the experimental datasets to stable. I'm not entirely sure if we would keep this for certain datasets. Why is it necessary?

Reply:
Didn't know it wasn't necessary, noticed that the tests verify that the metadata is correct...which is great.
I removed the lines item.